presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveTableStatistics.java

```diff
@@ -758,6 +759,31 @@ public void testAnalyzeForTablePartitionedByBigint()
                 row(null, null, null, null, 5.0, null, null));
     }
 
+    @Test
+    @Requires({NationPartitionedByBigintTable.class, UnpartitionedNationTable.class})
+    public void testEstBytesScannedAcrossJoinedTables()
+    {
+        String tableName1InDatabase = mutableTablesState().get(NATION.getName()).getNameInDatabase();
```

Comment:
can you have two tests for this, one per each table. It looks like these tables are not used together. How do they relate?